src/features/layer_features_panel/readme.md

```
@@ -0,0 +1,73 @@
+```
+
+### How it works
+
+1. `fetchLayerFeaturesResource` is fetching the data from the layer features endpoint (`/layers/${layerId}/items/search`).
+
+- New data is fetched each time the Selected area changes.
+- The data is only fetched if both are true:
+  - The Selected area is not empty.
+  - The associated layer is enabled (if the layer is required to be enabled).
+
+2. [layerFeaturesCollectionAtom](./atoms/layerFeaturesCollectionAtom.ts) transforms the data from `fetchLayerFeaturesResource` into an array of items (`FeatureCardCfg[]`)
```

Markdownlint

Expected: 1; Actual: 2; Style: 1/2/3
Ordered list item prefix

(MD029, ol-prefix)